.github/workflows/compatibilityCheck.yaml

```diff
@@ -0,0 +1,147 @@
+name: Compatibility Check
+
+on:
+  workflow_dispatch:
+    inputs:
+      mavenRepository:
+        description: 'URL of a Maven repository that holds dependencies to confirm'
+        required: true
+        type: string
+      mavenDependencies:
+        description: 'Key-value pairs of dependencies for Java'
+        required: true
+        type: string
+        
+  schedule:
+    - cron: '05 8 * * *' # 08:00 UTC every day
+
+jobs:
+  integrationTests:
+    name: Integration Tests for Compatibility Check
+    if: github.repository == 'suztomo/spring-cloud-gcp'
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        it:
+          - bigquery
```

Comment:
Is it intentionally only for 3 products for now?

Reply:
I just brought the rest back.